distribution/test/itests/test-itests-common/src/main/java/ddf/common/test/catalog/CatalogTestCommons.java

```diff
@@ -0,0 +1,164 @@
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.common.test.catalog;
+
+import static org.hamcrest.Matchers.equalTo;
+import static com.jayway.restassured.RestAssured.delete;
+import static com.jayway.restassured.RestAssured.given;
+
+import static ddf.common.test.AbstractIntegrationTest.CSW_PATH;
+import static ddf.common.test.AbstractIntegrationTest.REST_PATH;
+import static ddf.common.test.AbstractIntegrationTest.getFileContent;
+import static ddf.common.test.csw.CswTestCommons.getCswInsert;
+import static ddf.common.test.csw.CswTestCommons.getMetacardIdFromCswInsertResponse;
+import java.io.IOException;
+import java.util.Map;
+
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.xml.xpath.XPathExpressionException;
+
+import org.apache.http.HttpStatus;
+
+import com.jayway.restassured.response.Response;
+import com.jayway.restassured.response.ValidatableResponse;
+
+import ddf.common.test.AbstractIntegrationTest;
+
+public class CatalogTestCommons {
+
+    private static final String GEOJSON_NEAR_METACARD = "GeoJson near";
+
+    private static final String GEOJSON_FAR_METACARD = "GeoJson far";
+
+    private static final String PLAINXML_NEAR_METACARD = "PlainXml near";
+
+    private static final String PLAINXML_FAR_METACARD = "PlainXml far";
+
+    private static final String CSW_RESOURCE_ROOT = "/TestSpatial/";
+
+    private static final String CSW_METACARD = "CswRecord.xml";
+
+    public static String ingest(String data, String mimeType) {
+
+        return given().body(data)
+                .header(HttpHeaders.CONTENT_TYPE, mimeType)
+                .expect()
+                .log()
+                .all()
+                .statusCode(HttpStatus.SC_CREATED)
+                .when()
+                .post(REST_PATH.getUrl())
+                .getHeader("id");
+    }
+
+    public static String ingest(String data, String mimeType, boolean checkResponse) {
+        if (checkResponse) {
+            return ingest(data, mimeType);
+        } else {
+            return given().body(data)
+                    .header(HttpHeaders.CONTENT_TYPE, mimeType)
+                    .when()
+                    .post(REST_PATH.getUrl())
+                    .getHeader("id");
+        }
+    }
+
+    public static String ingestGeoJson(String json) {
+        return ingest(json, "application/json");
+    }
+
+    public static Map<String, String> ingestMetacards(Map<String, String> metacardsIds) {
```

Comment:
I'd personally like to see this method ripped apart, I'd prefer to address this is in the follow up ticket I mentioned, DDF-2492.